cypress/integration/tests/developer/applicationinventory/assessment/import.test.ts

```diff
@@ -142,23 +127,19 @@ describe("Application import operations", () => {
             cy.wait("@getApplication");
 
             // Import csv with non-existent businsess service and tag rows
-            const fileName = "non_existing_tags_business_service_rows.csv";
-            importApplication(filePath + fileName);
+            const fileName = "missing_tags_21.csv";
```

Comment:
what about missing BS test? Don't we need it anymore ?

Reply:
That was an oversight. I've fixed it.